R/common.R

```diff
@@ -122,6 +123,27 @@ runJaspResults <- function(name, title, dataKey, options, stateKey, functionCall
   }
 }
 
+registerFonts <- function() {
+
+  if (requireNamespace("ragg") && requireNamespace("systemfonts")) {
+
+    # # Not sure if we actually want to do this
+    # font_name0 <- "FreeSansJASP"
+    # font_name1 <- "LatoJASP"
+    # # TODO: this obviously only works on my machine
+    # font_file0 <- "~/github/jasp-desktop/Desktop/resources/fonts/FreeSans.ttf"
+    # font_file1 <- "~/github/jasp-desktop/Desktop/html/font/Lato-Regular.ttf"
+    #
+    # systemfonts::register_font(font_name0, normalizePath(font_file0))
+    # systemfonts::register_font(font_name1, normalizePath(font_file1))
+
+    print("registered FreeSansJASP & LatoJASP")
+    jaspGraphs::setGraphOption("family", .resultsFont)
```

Comment:
Where does `.resultsFont` come from?

Reply:
From the other PR, I see.

Reply:
yes. We could also pass this as an argument but there's not a very clear benefit and it would add yet another argument to `runJaspResults`.